phpcs.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<ruleset name="WordPress Coding Standards for Plugins">
+	<description>Generally-applicable sniffs for WordPress plugins</description>
+
+	<rule ref="WordPress-Core" />
+	<rule ref="WordPress-Docs" />
+
+	<!-- Whitelist p4 test classes for filename sniffs. -->
+	<rule ref="WordPress.Files.FileName">
+		<properties>
+			<property name="custom_test_class_whitelist" type="array" value="P4_TestCase,P4MasterThemeTest,OpenGraphTest,CustomTaxonomyTest,CustomPermalinksTest,CategoryPageTest"/>
```

Comment:
No need to add all the class names. Just the class that they extend. This should be enough.
	<!-- Whitelist p4 test classes for filename sniffs. -->

	<rule ref="WordPress.Files.FileName">
		<properties>
			<property name="custom_test_class_whitelist" type="array" value="P4_UnitTestCase"/>
		</properties>
	</rule>`

[custom unit test classes](https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/wiki/Customizable-sniff-properties#custom-unit-test-classes)

Also there is already a phpcs configuration file in repo (phpcs.xml.dist), please add this rule there

Reply:
Ahhh, that's much better! Will do.